src/hooks/use-key-press.ts

```diff
@@ -0,0 +1,44 @@
+import { useEffect } from 'react';
```

Comment:
This is really dope!! Ima have to add this to the study lesson, haha

Reply:
that would be so cool. it's pretty barebones right now, so depending on what you need, maybe allow it to take an array of keys or an object of keys to callbacks.

Reply:
Yeah, an object of keys to callbacks sounds ideal. We can always figure it out later, haha